vault/request_handling.go

```diff
@@ -320,6 +320,8 @@ func (c *Core) checkToken(ctx context.Context, req *logical.Request, unauth bool
 		case logical.ErrUnsupportedPath:
 			// fail later via bad path to avoid confusing items in the log
 			checkExists = false
+		case logical.ErrRelativePath:
```

Comment:
Is this a UserError (line 332 below), and if not, is it ok to return?

Reply:
Ah, I think you're right! Thanks for calling that out. I addressed this in https://github.com/hashicorp/vault/pull/14328/commits/c1e527d4886939aaded77c1de010fc62b3d20eaf.